torch/_higher_order_ops/_invoke_quant.py

```diff
@@ -0,0 +1,64 @@
+# mypy: allow-untyped-defs
+# need to fix prim_hop_base type annotations first
+
+import dataclasses
+
+import torch
+from torch._higher_order_ops.prim_hop_base import PrimHOPBase
+
+
+class InvokeQuantTracer(PrimHOPBase):
+    def __init__(self) -> None:
+        super().__init__("invoke_quant_packed")
+
+    def __call__(self, subgraph, operands, *, scheme=None, quant_options=None):
+        return super().__call__(
+            subgraph, operands, scheme=scheme, quant_options=quant_options
+        )
+
+
+invoke_quant_packed = InvokeQuantTracer()
```

Comment:
You're probably going to come across a test that asks you to add HOP db tests for each of the new HOPs in this pr (https://github.com/pytorch/pytorch/blob/a8ef423fed46bd0277830527dcdc83d4165ab095/torch/testing/_internal/hop_db.py#L61). It would be nice to add them (instead of bypassing them by adding them to the allowlist)